modules/config/default.js

```diff
@@ -43,6 +43,12 @@ export const settings = {
   groupActionsPosition: 'topRight', // oneOf [topLeft, topCenter, topRight, bottomLeft, bottomCenter, bottomRight]
   setOpOnChangeField: ['keep', 'default'], // 'default' (default if present), 'keep' (keep prev from last field), 'first', 'none'
 
+  convertableWidgets: {
```

Comment:
Value in number widget can be preserved after selecting slider widget (if value is valid and preserving values is enabled in config)